decls/i18n.js

```diff
@@ -85,6 +85,8 @@ declare interface I18n {
   set fallbackLocale (locale: Locale): void,
   get messages (): LocaleMessages,
   get dateTimeFormats (): DateTimeFormats,
+  get numberFormats (): NumberFormats,
```

Comment:
Note) 
I thinks `numberFormats` is missing so I fixed, though it is out of scope of #193...
I can make a separate PR if it is preferrable.

Reply:
I hope you will separate it as PR. 🙏

Reply:
Fixed!